src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs

```diff
@@ -112,8 +109,8 @@ public static ControlFlowGraph Create(IOperation body, ControlFlowGraph.Region e
             CheckUnresolvedBranches(blocks, builder._labeledBlocks);
             Pack(blocks, root, builder._regionMap);
             var methods = ArrayBuilder<IMethodSymbol>.GetInstance();
-            var methodsMap = ImmutableDictionary.CreateBuilder<IMethodSymbol, (ControlFlowGraph.Region, IOperation, int)>();
```

Comment:
>var ordinalToBlockMap = PooledDictionary<int, FlowAnalysis.BasicBlock>.GetInstance(); [](start = 12, length = 85)

It feels like the ```builder``` should be enough to map an ordinal to an instance. I'll follow up on that myself